zips/zip-0325.md

```diff
@@ -78,16 +78,16 @@ tree:
 The tree has the following general structure, specified in more detail below:
 
 - $m_{\mathsf{metadata}}$: Metadata Key tree
-  - $m_{\mathsf{metadata}} / 325' / \mathsf{coinType}' / \mathsf{account}'$ - Account Metadata Key
-    - $\ldots / 0'$ - Account-level Inherent Metadata Key
+   - $m_{\mathsf{metadata}} / 325' / \mathsf{coinType}' / \mathsf{account}'$ - Account Metadata Key
+      - $\ldots / 0'$ - Account-level Inherent Metadata Key
+         - $\ldots / \ldots$ - (Reserved for future updates to this ZIP)
+         - $\ldots / (\mathtt{0x7FFFFFFF}', \mathsf{PrivateUseSubject})$ - Private-use Inherent Metadata Key
+      - $\ldots / 1'$ - Account-level External Metadata Key
+         - $\ldots / (0', \mathsf{FVKTypedItem})$ - Imported UFVK Metadata Key
+            - $\ldots / \ldots$ - (Reserved for future updates to this ZIP)
+            - $\ldots / (\mathtt{0x7FFFFFFF}', \mathsf{PrivateUseSubject})$ - Private-use External Metadata Key
+         - $\ldots / \ldots$ - (Reserved for future updates to this ZIP)
```

Comment:
MultiMarkdown is picky about the number of spaces used to indent nested lists: it must be at least 3.